db/migrate/20240705082709_create_mail_delivery_failures.rb

```diff
@@ -0,0 +1,11 @@
+class CreateMailDeliveryFailures < ActiveRecord::Migration[7.1]
+  def change
+    create_table :mail_delivery_failures do |t|
+      t.string :email_address
+      t.string :mailer_class
+      t.string :mailer_action_method
```

Comment:
I think we'll want these to all be `null: false` to ensure they can't be set to `NULL`, as we'd expect them always to be set.